### ISSUE TITLE
Ajax запросы

### ISSUE THREAD

**Issue by Ananaskelly** [malinink/games#112]
Родительская задача #
Исполнитель @

Описание
Реализация ajax запросов клиента
- [запрос на получение токена](https://github.com/malinink/games/wiki/Protocol#%D0%97%D0%B0%D0%BF%D1%80%D0%BE%D1%81-%D0%BD%D0%B0-%D0%BF%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D0%B8%D0%B5-%D1%82%D0%BE%D0%BA%D0%B5%D0%BD%D0%B0--%D0%9F%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D0%B8%D0%B5-%D1%82%D0%BE%D0%BA%D0%B5%D0%BD%D0%B0)
- [передача хода](https://github.com/malinink/games/wiki/Protocol#%D0%A5%D0%BE%D0%B4-%D0%B8%D0%B3%D1%80%D0%BE%D0%BA%D0%B0--%D0%9E%D1%82%D1%87%D0%B5%D1%82-%D0%BE-%D1%81%D0%BE%D0%B2%D0%B5%D1%80%D1%88%D0%B5%D0%BD%D0%BD%D0%BE%D0%BC-%D1%85%D0%BE%D0%B4%D0%B5)

Теги: task

**Comment by Ananaskelly** [comment]
а как это правильнее реализовать? Вернее где, создать отдельный .js и там функции?

**Comment by malinink** [comment]
Да, я бы сделал отдельный компонент (класс) - управляющий, и по аналогии с бекендом реализовал бы в нем проверки и реакцию на получение информации, ну и их обработку.

Т.е. один главный - который следит за соединением с WS. И его поддержкой. 
И куча небольших классов по одному для каждого протокола. Которые будут вызываться из главного.

Реализация - наверное с точки зрения js, сделал бы всё в одном файле.
В принципе можно и разбить, если есть желание и потом каким нибудь [requirejs](http://requirejs.org/) их подключать
